File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/VisualBasic/1049/WebService/CodeBehind.vb
Imports System.Web
Imports System.Web.Services
Imports System.Web.Services.Protocols

$if$ ($targetframeworkversion$ >= 3.5)' Чтобы разрешить вызывать веб-службу из скрипта с помощью ASP.NET AJAX, раскомментируйте следующую строку.
' <System.Web.Script.Services.ScriptService()> _
$endif$<WebService(Namespace:="http://tempuri.org/")> _
<WebServiceBinding(ConformsTo:=WsiProfiles.BasicProfile1_1)> _
<Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()> _
Public Class $codebehindclassname$
     Inherits System.Web.Services.WebService

    <WebMethod()> _
    Public Function HelloWorld() As String
        Return "Привет всем!"
    End Function

End Class

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/CSharp/1049/WebService/CodeBehind.cs
using System;
using System.Collections.Generic;
$if$ ($targetframeworkversion$ >= 3.5)using System.Linq;
$endif$using System.Web;
using System.Web.Services;

/// <summary>
/// Сводное описание для $codebehindclassname$
/// </summary>
[WebService(Namespace = "http://tempuri.org/")]
[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
$if$ ($targetframeworkversion$ >= 3.5)// Чтобы разрешить вызывать веб-службу из скрипта с помощью ASP.NET AJAX, раскомментируйте следующую строку. 
// [System.Web.Script.Services.ScriptService]
$endif$public class $codebehindclassname$ : System.Web.Services.WebService {

    public $codebehindclassname$ () {

        //Раскомментируйте следующую строку в случае использования сконструированных компонентов 
        //InitializeComponent(); 
    }

    [WebMethod]
    public string HelloWorld() {
        return "Привет всем!";
    }
    
}
